packages/block-library/src/navigation/style.scss

```diff
@@ -1,7 +1,20 @@
-.wp-block-navigation .wp-block-navigation__container {
+.wp-block-navigation {
+	// Normalize list styles.
+	ul,
+	ul li {
+		list-style: none;
+
+		// Overrides generic ".entry-content li" styles on the front end.
```

Comment:
These styles were previously in the theme.scss stylesheet, but are structural so I moved them here.

I also simplified them and tweaked their specificity, which is still pretty low. I did verify they overrode generic `.entry-content li` margin rules a theme might add.